docs/api-references/uploadimage.md

```diff
@@ -0,0 +1,67 @@
+### Upload Image API
+#### `POST` Image
+
+```
+/api/users/upload-image
+```
+**Upload image**
+
+##### Request header
+| Key | Value |
+|---|---|
+| Accept | application/json |
+| Content-Type | image/* |
+| Authorization | {token_type} {access_token} |
+
+##### Request parameter
+| Key | Description |
+|---|---|
+| file |  A binary file (up to 10Mb) |
+
+##### Response parameter
+
+| Key | Type | Description |
+|---|---|---|
+| fileName | string | the name of image |
+
+##### Sample Response the name of file save success
+```json
+{
+  "imageName": "1505789329number-1.png"
```

Comment:
Should be used `snake_case`.

Reply:
In the same api, if request has file -> upload image to server, if request has fileName -> remove the file há name = fileName on server